### PR TITLE
Add load balancer resource

### PR DESCRIPTION
**Pull request by MusicDin**
This PR adds LXD load balancer resource.

Load balancer only works with OVN networks, therefore, OVN is also configured before the tests are run (in GH actions).

### CODE REVIEW

File: lxd/resource_lxd_network_lb.go [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+func resourceLxdNetworkLB() *schema.Resource {
+	return &schema.Resource{
+		Read:   resourceLxdNetworkLBRead,
+		Create: resourceLxdNetworkLBCreate,
+		Update: resourceLxdNetworkLBUpdate,
+		Delete: resourceLxdNetworkLBDelete,
+		Exists: resourceLxdNetworkLBExists,
+
+		Schema: map[string]*schema.Schema{
+			"remote": {
```
[the remark this answers is not in the document]
Comment (MusicDin) [reply lxd/resource_lxd_network_lb.go:21]: Thanks. I'm not sure load balancers can be associated with any project at all. I have to verify, and either add missing field or remove existing project logic.

[the remark this answers is not in the document]
Comment (MusicDin) [reply lxd/resource_lxd_network_lb.go:21]: My bad, I had a typo in `features.networks` option when testing project out. I have added the missing attribute. Thanks again :)

**Review by roosterfish — APPROVED** [review]
LGTM!

**Review by simondeziel — APPROVED** [review]
Awesome, thanks you!

**Review by adamcstephens — APPROVED** [review]

**Pull request merged**